references/cuegen/testdata/valid/exported.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2023 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+type A struct {
+	Field1 string `json:"field1"`
+	field2 string
```

Comment:
The field without json tag won't be exported to cue makes sense to me, so for some specific type like `unstructered`, we can register the type as `any`, great job!

Reply:
The field here is ignored because it is not exported. If an exported `FieldName` does not have a json tag, it will still be written to the cue with `FieldName`